quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/TruststorePropertyMappers.java

```diff
@@ -0,0 +1,21 @@
+package org.keycloak.quarkus.runtime.configuration.mappers;
+
+import org.keycloak.config.TruststoreOptions;
+
+import static org.keycloak.quarkus.runtime.configuration.mappers.PropertyMapper.fromOption;
+
+public class TruststorePropertyMappers {
```

Comment:
I noticed we're missing the `TruststorePropertyMappers` from the map of all mappers here:
https://github.com/keycloak/keycloak/blob/71b6757c2f7a3e08b857944bf9416f93602318a1/quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/PropertyMappers.java#L26-L43

Reply:
Added.  @pedroigor mentioned we could get an issue started on simplifying / consolidating some of the config-api / mapper logic - maybe we can think of ways to eliminate some of the manual registration logic.